default-config/config

```diff
@@ -109,11 +109,14 @@ AddToFunc MoveToCurrent
 # Function: ViewManPage $0
 #
 # This function loads the man page $0 in an terminal
-# and is used with the help menu.
+# and is used with the help menu. If the manual page is
+# not found, point users at fvwm.org and keep terminal open.
 DestroyFunc ViewManPage
 AddToFunc   ViewManPage
 + I Exec exec $[infostore.terminal] -g 80x40 \
-  -n "Manual Page - $0" -T "Manual Page - $0" -e man "$0"
+  -n "Manual Page - $0" -T "Manual Page - $0" -e 'man "$0" || \
+  (echo "$[gt.Documenation can be found at https://www.fvwm.org]"; \
```

Comment:
Does this work as intended?  Shouldn't it be:

```
$[gt.Documenation] can be found at https://www.fvwm.org
```

Also, I'm presuming you meant `gt.Documentation` and not `gt.Documenation`?

Reply:
I want to translate the whole string, in my tests the string returns just fine, I don't have any translations to test on that side. Though I guess there is a spelling error there. I'll update the pot file too.

Note, probably pointless here since all the documentation is not translated. I just added that in case some translator wanted to translate the default-config.